modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -49,6 +50,11 @@ The following values for `operatorLogLevel` are supported:
 |Defines the Proxy to be used when calling master API
 and upstream registries.
 
+|`affinity`
+|You can use the `affinity` configuration parameter to configure pod scheduling preferences and constraints for Image Registry Operator pods.
+
+You can configure affinity settings to use the `podAffinity` or `podAntiAffinity` spec. Both options can use either `preferredDuringSchedulingIgnoredDuringExecution` rules or `requiredDuringSchedulingIgnoredDuringExecution` rules. If not specified, a preferred anti-affinity rule is applied by default, and a required anti-affinity rule is applied if `spec.replicas` is set to `2`.
```

Comment:
> If not specified, a preferred anti-affinity rule is applied by default [...]

Hi @abrennan89 - how did you observe this default preferred anti-affinity? I'm asking because I couldn't find it in the code base (I haven't looked at a running instance of the registry though!).

Reply:
Hey @flavianmissi 🙂 
Tbh I can't remember where this came from because I worked on these docs a few weeks ago. I didn't have draft docs, so everything here came from just doing research in existing k8s / OCP docs online IIRC. I can take out the part about the default rule - do you know if there is any default settings we should talk about here, or should we just leave it as the first 2 lines?

Reply:
No problem, @abrennan89! I'll launch a cluster and double check what we have for defaults and let you know.

Reply:
With two replicas, it looks as expected:
```
    spec:
      affinity:
        podAntiAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            - labelSelector:
                matchLabels:
                  docker-registry: default
              namespaces:
                - openshift-image-registry
              topologyKey: kubernetes.io/hostname
```
With 3 replicas there is no affinity - here's the full deployment: https://gist.github.com/flavianmissi/4ead2cf38a8a015b4b0462dcd120b95a

Reply:
Awesome, thank you so much!